### PR TITLE
feat/portal

### PR DESCRIPTION
**Pull request by wasinphand**

### CODE REVIEW

File: pages/portal/box.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { PortalBox } from '../../interfaces/box';
+import Button from '../../commons/components/component.button';
+
+const Box: React.FC<PortalBox> = props => {
+  const { name } = props;
+  return (
+    <div>
+      <Button onClick={() => {}} type='button' className=''>
```
Comment (hamzaabamboo) [inline pages/portal/box.tsx]: I think adding onClick as props would be nice
``` 
const { name, onClick = () => {} } = props;
```
```
export interface PortalBox {
  name: string;
  onClick?: () => void;
}
```

File: pages/schedule/agenda.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import compareAsc from 'date-fns/compareAsc';
+
+interface Agenda {
```
Comment (hamzaabamboo) [inline pages/schedule/agenda.tsx]: `AgendaItem[]` make more sense. since Agenda is collection of AgendaItems

File: .vscode/settings.json [hunk 3]
```diff
@@ -10,5 +10,6 @@
     "source.fixAll.eslint": true
   },
   // "editor.formatOnSave": true,
-  "editor.tabSize": 2
+  "editor.tabSize": 2,
+  "editor.formatOnSave": true
```
Comment (new5558) [inline .vscode/settings.json:13]: So many problem with this line. will gitignore this file

File: tailwind.config.js [hunk 4]
```diff
@@ -1,6 +1,12 @@
 module.exports = {
   theme: {
-    extend: {}
+    extend: {},
+    spacing:{
```
Comment (new5558) [inline tailwind.config.js]: we will deal with this later.

**Pull request closed without merging**